### PR TITLE
upgrade: docusaurusV2

### PR DESCRIPTION
**Pull request by KishaniKandasamy**
Changes:
upgrade: docusaurusV2

Screenshots of the change:
Screenshot (19)

### CODE REVIEW

**Comment by KishaniKandasamy** [comment]
Screenshot (17)
I'm stuck at this point  as **CompLibrary is deprecated in v2**. Therefore when I move pages/en files in v1 to src/pages in v2, It shows errror like "Cannot find module: CompLibrary".

**Comment by juzhiyuan** [comment]
Great! We will review recently :)

File: website2.0/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  },
+  "browserslist": {
+    "production": [">0.5%", "not dead", "not op_mini all"],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
+  }
+}
```
Comment (juzhiyuan) [inline website2.0/package.json:30]: We need a new line here :)

File: website2.0/static/css/custom.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
+/**
```
Comment (juzhiyuan) [inline website2.0/static/css/custom.css:1]: After we migrated all needed files, we need to add ASF header here.

File: .gitignore [hunk 3]
```diff
@@ -1,12 +1,28 @@
+# dependencies
+website2.0/node_modules
```
Comment (juzhiyuan) [inline .gitignore:2]: Hi, could we omit the wesite2.0 directory? Set all needed files under the root directory directly is fine.

**Comment by juzhiyuan** [comment]
I see there has a cli to help us migrate.

https://v2.docusaurus.io/docs/migration/automated

**Comment by KishaniKandasamy** [comment]
> I see there has a cli to help us migrate.
> 
> https://v2.docusaurus.io/docs/migration/automated

Yes. Initially I did  running the migration cli, and tried to complete the missing parts  using  the manual migration process.

# migration command format
[1]npx @docusaurus/migrate migrate ( v1 website directory)  (desired v2 website directory)
Ex: npx @docusaurus/migrate migrate ./v1-website ./v2-website

That's why I created website2.0 :)
Screenshot (20)

But here It says **some parts can't easily be automated, and  have to fallback to the manual process.**

File: website2.0/package.json (hunk 1 above)
Comment (KishaniKandasamy) [reply website2.0/package.json:30]: Yes .I will correct it :)

File: website2.0/static/css/custom.css (hunk 2 above)
Comment (KishaniKandasamy) [reply website2.0/static/css/custom.css:1]: yes Juzhiyuan

**Comment by juzhiyuan** [comment]
ok, just to make sure, this PR is created by the following steps:

1. using the command to auto migrate;
2. fix missing parts which that command cannot do.

File: .gitignore (hunk 3 above)
Comment (KishaniKandasamy) [reply .gitignore:2]: Yes Juzhiyuan .It would be better to under have the root directory   .

**Comment by KishaniKandasamy** [comment]
> ok, just to make sure, this PR is created by the following steps:
> 
> 1. using the command to auto migrate;
> 2. fix missing parts which that command cannot do.

Yes

**Comment by KishaniKandasamy** [comment]
Here the problem is Some parts of the migration can't be automated (**particularly the pages**), so we will have to migrate them manually.
Also  CompLibrary is deprecated in v2. Therefore when I move pages/en files in v1 to src/pages in v2, It shows errror like "Cannot find module: CompLibrary".
The guide says to write our own React component or use Infima styles.So **First Do we need to modify the  pages according to this** ?
Screenshot (22)

**Comment by juzhiyuan** [comment]
Hi @KishaniKandasamy, I would prefer using Infima styles, but which one is easier for you, then we use it.

**Comment by KishaniKandasamy** [comment]
> Hi @KishaniKandasamy, I would prefer using Infima styles, but which one is easier for you, then we use it.

Hi Zhiyuan , Yes. If you give me sometime, I could  solve this using Infima styles. as I am not most  familiar with Infima styles.

**Comment by juzhiyuan** [comment]
Sure :)

**Pull request closed without merging**